repository/post.go

```diff
@@ -0,0 +1,95 @@
+package repository
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Masterminds/squirrel"
+	"github.com/georgysavva/scany/pgxscan"
+	"github.com/serhiihuberniuk/blog-api/models"
+)
+
+func (r *Repository) CreatePost(ctx context.Context, post *models.Post) error {
+	sql := "INSERT INTO posts VALUES ($1,$2,$3,$4,$5,$6)"
```

Comment:
please rewrite all insert queries to `INSERT INTO table(column1, column2 .... ) VALUES (value1, value2 ....)`